cli/tests/integrations/test_package.py

```diff
@@ -207,7 +207,8 @@ def test_describe():
     "framework"
   ],
   "versions": [
-    "0.8.1"
+    "0.8.1",
+    "0.9.0-RC3"
```

Comment:
I see. Maybe the issues is in how we generate the version in the CLI. Is the plan for the user to maybe do something like `dcos package install --version=0.8.1 marathon`? or `dcos package install --version=0 marathon`? @ConnorDoyle?